GoogleSignIn/Sources/Public/GoogleSignIn/GIDGoogleUser.h

```diff
@@ -76,14 +76,14 @@ NS_ASSUME_NONNULL_BEGIN
 ///
 /// @param completion A completion block that takes a `GIDGoogleUser` or an error if the attempt to
 ///     refresh tokens was unsuccessful.  The block will be called asynchronously on the main queue.
-- (void)doWithFreshTokens:(void (^)(GIDGoogleUser *_Nullable user,
-                                    NSError *_Nullable error))completion;
+- (void)refreshTokensWithCompletion:(void (^)(GIDGoogleUser *_Nullable user,
```

Comment:
We may want to indicate (via the method name) that a token refresh will only be performed when needed.

Reply:
`refreshTokensIfNeddedWithCompletion`? Do you have a better name on your mind?

Reply:
I think that works.  We'll have a chance to review our naming before release.